README.md

```diff
@@ -230,16 +263,8 @@ Failed Login	hello	7 minutes ago	N/A	my awesome app
 
 ## Customization
 
-The authenticator of the CLI defaults to the default Auth0 cloud `auth0.auth0.com`. This can be customized for personalized cloud offerings by setting the following env variables:
```

Comment:
I preferred to move this part to the login section instead of having this under customization.